Trees/BreadthFirstTreeTraversal.js

```diff
@@ -17,7 +17,27 @@ class BinaryTree {
     this.traversal = []
   }
 
-  breadthFirst () {
+  breadthFirstIterative () {
+    this.traversal = []
+    if (this.root) {
+      this.traversal.push(this.root)
+    }
+    let currentSize = this.traversal.length
+    for (let i = 0; i < currentSize; i++) {
+      if (this.traversal[i].left) {
```

Comment:
Could shorten the body by doing `let currentNode = traversal[i]`

Reply:
Haha .. That's actually how I had it before making my final changes.

I ultimately decided I'd rather save 1 line of code while sacrificing some width of each line. My motivation was keeping the algo as short as possible since it wasn't going to be that complicated to begin with.

Not a huge deal though .. I can change it back.